<launch>

    <node pkg="sensors" type="imu" name="imu" />

    <node pkg="sensors" type="backstepping_controller.py" name="backstepping_controller" />

    <node pkg="sensors" type="motors.py" name="motors" />

    <node pkg="sensors" type="cs_ch_3.py" name="cs_ch_3" />

    <!--Launch rosbag to record topics -->
    <node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/ubuntu/catkin_ws/src/sensors/bags/SpeedAndHeading/0,3_speed_ref_heading" />

</launch>
